### PR TITLE
Update D06 - Protect Secrets.md

### PR DESCRIPTION
**Pull request by Sascha-Block**
Initial KickOff filling D06 - Protect Secrets with life in kind of first Threat Scenarios and suggestions for How to prevent...

### CODE REVIEW

File: D06 - Protect Secrets.md [hunk 1]
```diff
@@ -2,8 +2,15 @@
 
 
 ## Threat Scenarios
+- in worst Case Secrets would be stored somewhere in our Container Environment
```
Comment (drwetter) [inline D06 - Protect Secrets.md]: Definition of somewehre?

File: D06 - Protect Secrets.md [hunk 2]
```diff
@@ -2,8 +2,15 @@
...
+- exposed environment variables 
+- priviliged access, in worst case as root user
```
Comment (drwetter) [inline D06 - Protect Secrets.md:7]: --verbose?

File: D06 - Protect Secrets.md [hunk 3]
```diff
@@ -2,8 +2,15 @@
...
 
 ## How Do I prevent?
+- running containers as a non-root user
```
Comment (drwetter) [inline D06 - Protect Secrets.md:10]: This has nothing to do with it

File: D06 - Protect Secrets.md [hunk 4]
```diff
@@ -2,8 +2,15 @@
...
+- using multi-stage builds
```
Comment (drwetter) [inline D06 - Protect Secrets.md:11]: This probably has also nothing to do with it unless you have in the first stage a credential which is futile, right?

**Review by Sascha-Block — COMMENTED** [review]
Update definition of somewhere

**Comment by drwetter** [comment]
Hi Sascha,

I rather like to close this. Please try again if you still want.
Prerequisites:

* If it is not production quality please PR against the <BULLETPOINT>_DEV branch which I'll create later
* full sentences and explanations will be always appreciated instead of bullet-points which are often not self-explanatory
* Threat scenarios are describing points from an attackers view or describe the value for an attacker. So why do you think the points you're mentioned are important for the attacker?

**Pull request closed without merging**